redisinsight/api/provider-interfaces/models/client-certificate.ts

```diff
@@ -0,0 +1,12 @@
+// todo: discuss
+// probably not needed for cloud
```

Comment:
The client cert is stored in the Redis Cloud SM database so we can retrieve it from there.
We will need the model, but in the provider we only need the `get` which will return the same Client Certificate for the specific database.

So we need a way to identify the database associated with the client certificate. 
This can be done by piggybacking on the `id` property (i.e. `ClientCertificate.id` will hold the `databaseId`), or we can add an additional `databaseId` property.

WDYT ?